hass_nabucasa/__init__.py

```diff
@@ -44,6 +50,15 @@
 from .voice import Voice
 from .voice_api import VoiceApi
 
+__all__ = [
+    "AlreadyConnectedError",
+    "Cloud",
+    "CloudError",
```

Comment:
Maybe add `CloudClient` which should be implemented by users of the library?

Reply:
Added a few others as well to promote imports from the pacakge root.